launchable/test_runners/googletest.py

```diff
@@ -0,0 +1,46 @@
+import sys
+import re
+import click
+
+from . import launchable
+from ..testpath import TestPath
+
+
+def make_test_path(cls, case) -> TestPath:
+    return [{'type': 'class', 'name': cls}, {'type': 'case', 'name': case}]
```

Comment:
Is this path representation consistent with `CaseEvent.default_path_builder`? It doesn't look like it is to meas this uses type=case but the `default_path_builder` uses `type=testcase`.

I've said this before, but I think we need to rework the test path construction code in the report collection side given that two sides need to be aligned on the paths they create. Too many different test runners relying on the particular behavior of `default_path_builder` makes it really fragile. Aiming for post 1.0.